dns/checkip/src/opnsense/mvc/app/models/OPNsense/DynDNS/CheckIP_Service.xml

```diff
@@ -0,0 +1,68 @@
+<model>
+   <mount>//OPNsense/DynDNS/CheckIP</mount>
+   <version>1.0.0</version>
+   <description>CheckIP settings - Service</description>
+   <items>
+      <service type="ArrayField">
+         <default type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+         </default>
+         <name type="TextField">
+            <default></default>
+            <Required>Y</Required>
+            <mask>/^([a-zA-Z0-9_]){0,255}$/u</mask>
+            <ValidationMessage>A valid service name is mandatory</ValidationMessage>
+         </name>
+         <url type="TextField">
```

Comment:
I have not used it but we do have an url field type:

https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/models/OPNsense/Base/FieldTypes/UrlField.php

Reply:
Thank you.  The url field type is much nicer.